package.json

```diff
@@ -14,54 +14,54 @@
     "test": "echo \"Write tests! -> https://gatsby.dev/unit-testing\" && exit 1"
   },
   "dependencies": {
-    "@fontsource/roboto": "^5.0.8",
-    "axios": "^1.6.2",
+    "@fontsource/roboto": "^5.0.12",
```

Comment:
In future, I'd recommend doing the package update on a separate PR. If something goes wrong, it becomes more difficult to determine the cause if we're packaging all the changes together.

Reply:
I'm wondering if the package updates were the reason the Netlify build was taking so long to complete.

We still have to create a release branch potentially from both #203 and this one, so I'm wondering if we:
- Put the menu change onto a new PR
- Put the package updates + hero.js change onto a new PR
- That way you can merge them one at a time onto develop and make sure they run on Netlify before creating a release branch from develop on Monday.

Reply:
I'm creating a develop Netlify environment, merging the first PR (#203) onto develop (see if it completes), then will merge #204 onto develop and see if it completes.

Reply:
@tqureshi-uog The develop branch did not finish within the time limit after merging #204. Would you be able to separate the PR into two PRs?

- One with the menu change
- One with the package updates + hero.js change

Then we can reset develop, merge #203, then merge the menu change and see if it completes without issue

Reply:
Actually...hang on. It looks like it almost finished. I'm going to re-run develop one more time (it's building the preview site)...maybe the cache will help it finish the next time it runs.

Reply:
@tqureshi-uog - Develop (with #203 and #204 merged) deployed successfully the second time around. 👍